Makefile

```diff
@@ -14,9 +14,10 @@ poetry-remove:
 #* Installation
 .PHONY: install
 install:
-	poetry lock -n && poetry export --without-hashes > requirements.txt
+	#poetry lock -n
```

Comment:
@skim0119 Why are these commented out?

Reply:
These commands are supposedly used by developers only, and there is no need to update the dependency check during installation.